public/test/core/redux/reducerTester.ts

```diff
@@ -18,6 +18,7 @@ export interface When<State> {
 export interface Then<State> {
   thenStateShouldEqual: (state: State) => When<State>;
   thenStatePredicateShouldEqual: (predicate: (resultingState: State) => boolean) => When<State>;
+  whenActionIsDispatched: (action: PayloadAction<any> | Action<any>) => Then<State>;
```

Comment:
Added this to this type so it's possible to chain multiple actions.

Reply:
Excellent addition, but you're not using it, right? In that case I would leave it as is.

Reply:
It's used in the test. `whenActionIsDispatched` returns `Then`, which doesnt have `whenActionIsDispatched` method, so if I want to call a few `whenActionIsDispatched` one on top another, I'll get a type error.

Reply:
Of course 🙈 , my bad